src/PortfolioPlanning/Components/Plan/PlanPage.tsx

```diff
@@ -144,7 +144,7 @@ export default class PlanPage extends React.Component<IPlanPageProps, IPortfolio
     };
 
     private _renderItemDetailsDialog = (): JSX.Element => {
-        if (this.props.selectedItem) {
+        if (this.props.selectedItem && this.props.selectedItem.canMove) {

```

Comment:
Ideally the whole context menu should be disabled while updating. I think you can pass disabled to the button instead.

Reply:
Yes, I didn't have your context menu change first, now I just pulled that change in and passed disabled.